runtime/client_http_request_test.go

```diff
@@ -192,7 +192,7 @@ func TestBarClientWithoutHeaders(t *testing.T) {
 
 	logs := gateway.AllLogs()
 
-	assert.Equal(t, 1, len(logs))
+	assert.Equal(t, 3, len(logs))
```

Comment:
we should consider putting a filter in all logs to see only logs by zanzibar. this count doesn't achieve much.

Reply:
I think removing these tests should be sufficient since they don't achieve much. Creating a filter would be difficult because we currently have no way of knowing where the logs came from

Reply:
A bonus from removing these tests is that we can pin tchannel at ^1.5.0 instead of ^1.6.0